src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1182,6 +1183,7 @@ private bool IsQueryMatch(string query, string symbolName)
             return symbolName.IndexOf(query, StringComparison.OrdinalIgnoreCase) >= 0;
         }
 
+        // https://microsoft.github.io/language-server-protocol/specification#textDocument_codeAction
```

Comment:
omg we should have comments like this all over... so great! (not in this PR :smile:  )